2-copy-of-code/lesson-13/scripts/amazon.js

```diff
@@ -98,8 +111,19 @@ document.querySelectorAll('.js-add-to-cart')
 
       addedMessage.classList.add('added-to-cart-visible');
 
-      setTimeout(() => {
+      // Check if there's a previous timeout for this
```

Comment:
@Harishprabhavathy Let's take timer 60s

At 1:00:00, you add an item to the cart.
A 60-second timer starts for the message to disappear.
At 1:00:30, you add the same item to the cart again.
The initial timer (which had 30 seconds left) should reset to 60 seconds because the item was added again.
When the same item is added to the cart again, the timer should reset to 60 seconds from the latest action time.
The previous timer should be removed, and the message should only disappear after the new/latest item added to the cart (60-second period)

You can observe the time difference in the previous code by adding the same item multiple times